examples/04-grav/plot_laguna_del_maule_inversion.py

```diff
@@ -25,14 +25,18 @@ def run(plotIt=True, cleanAfterRun=True):
 
     # Start by downloading files from the remote repository
     url = "https://storage.googleapis.com/simpeg/Chile_GRAV_4_Miller/"
-    cloudfiles = ['LdM_grav_obs.grv', 'LdM_mesh.mesh',
-                  'LdM_topo.topo', 'LdM_input_file.inp']
-
-    basePath = os.path.sep.join(os.path.abspath(os.getenv('HOME')).split
-                                (os.path.sep)+['Downloads']+['SimPEGtemp'])
-    basePath = os.path.abspath(remoteDownload(url,
-                                              cloudfiles,
-                                              basePath=basePath+os.path.sep))
+    cloudfiles = [
+        'LdM_grav_obs.grv', 'LdM_mesh.mesh',
+        'LdM_topo.topo', 'LdM_input_file.inp'
+    ]
+
+    # Download to Downloads/SimPEGtemp
```

Comment:
I think that it could be both? You can call `abspath` on the thing and it should expand it out regardless.

```python
folder='here'
folder='or/here'
folder='/Users/rowan/downloads/or/even/here'
```

I see all of these as folders?